.github/workflows/branch.yml

```
@@ -0,0 +1,16 @@
+name: nf-core branch protection
+# This workflow is triggered on PRs to master branch on the repository
+# It fails when someone tries to make a PR against the nf-core `master` branch instead of `dev`
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      # PRs are only ok if coming from an nf-core `dev` branch or a fork `patch` branch
+      - name: Check PRs
+        run: |
+          { [[ $(git remote get-url origin) == *atacseq ]] && [[ ${GITHUB_HEAD_REF} = "dev" ]]; } || [[ ${GITHUB_HEAD_REF} == "patch" ]]
```

shouldn't it be 
```suggestion
          { [[ $(git remote get-url origin) == *nf-core/atacseq ]] && [[ ${GITHUB_HEAD_REF} = "dev" ]]; } || [[ ${GITHUB_HEAD_REF} == "patch" ]]
```

otherwise it could come from a fork named the same way?

---

It should be - though if we don't allow these, it will always fail if the PR comes from such a fork. Which is also bad, e.g. when someone tries to contribute to fix a bug or something...

---

But that fix would have to go to `dev` right? And not directly to `master`?

---

Exactly, that's the protection to master branch, that it should come only from nf-core/atacseq dev

---

Aah fair point - didn't see that when first looking at it - I agree, then it should be nf-core/<pipelinename>:dev